<?xml version="1.0" encoding="utf-8"?>

<launch>

  <node name="cannon_decision_maker_py" output="screen" pkg="control" type="cannon_decision_maker.py">

    <param name="YAW_STEP"   type="int" value="2" /> 
    <param name="PITCH_STEP"   type="int" value="2" /> 
    <param name="RATE"   type="int" value="10" /> 

  </node>

</launch>